Source/FakeItEasy.Tests/Core/DefaultSutInitializerTests.cs

```diff
@@ -57,17 +57,27 @@ public void Should_use_same_instance_when_multiple_arguments_are_of_same_type()
         }
 
         [Test]
-        public void Should_pass_empty_fake_options_to_fake_manager()
+        public void Should_pass_impotent_options_builder_to_fake_manager_for_each_constructor_argument()

```

Comment:
Oh, and about this. @thomaslevesque, I speak English as my first (only) language and have what's probably a larger than average vocabulary.
If you're so inclined, do point out when I use terms (especially in code, comments, or documentation) that would give a non-native speaker difficulties.
For example, would the method name be better with "do-nothing" or "no-op" or "empty" in place of "impotent"? I can make such a change.